sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -33,7 +34,7 @@
 
 
 ###############################################################################
-###########################   Creating Surfaces   #############################
+#                            Creating Surfaces
 ###############################################################################
```

Comment:
I certainly have no problem with changing the output headers like this.  But is there a rationale behind this?  Just curious for consistency going forward.